source/faq/fundamentals.txt

```diff
@@ -41,35 +41,51 @@ the columns in a relational database table.
 How do I create a database and a collection?
 --------------------------------------------
 
+.. note:: 
+
+   You can enter the commands referenced in this FAQ by using 
+   the :doc:`Mongo Shell </mongo/>`. The MongoDB Shell is an 
+   interactive JavaScript interface to MongoDB. You can use the MongoDB 
```

Comment:
We need to be consistent in the terms here. We used "mongo Shell" above but are using MongoDB Shell here.